docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3'
+
+services:
+  #find a way to tear this down after the init scripts has worked
+  setup-datastore:
+    build: ./scripts/setup-datastore/
+    depends_on:
+      - datastore
+    environment:
+      MEILISEARCH_HOST: http://meilisearch:7700
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/key/creds.json
+      GCLOUD_PROJECT: pname
+      DATASTORE_EMULATOR_HOST: datastore:8000
+      DATASTORE_PROJECT_ID: pname
+
+  #find a way to tear this down after the init scripts has worked
+  setup-meilisearch:
+    build: ./tools/meilisearch-index-generator/
+    depends_on:
+      - setup-datastore
+      - meilisearch
+    environment:
+      MEILISEARCH_HOST: http://meilisearch:7700
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/key/creds.json
+      GCLOUD_PROJECT: pname
+      DATASTORE_EMULATOR_HOST: datastore:8000
+      DATASTORE_PROJECT_ID: pname
```

Comment:
setup-datastore_1 throws errors
> setup-datastore_1    | google.api_core.exceptions.ServiceUnavailable: 503 failed to connect to all addresses

Nice to have:
- Enabled cache when building images
- Python package installation while building the image